controllers/backup_it_utils_test.go

```diff
@@ -285,3 +289,21 @@ func (ts *BackupTestSuite) notifyObserverOfBackupJobStatusChange(status k8upObse
 	event.Event = status
 	observer.GetUpdateChannel() <- event
 }
+
+func (ts *BackupTestSuite) newBackupWithTags() *k8upv1.Backup {
+	BackupWithTags := ts.newBackup()
+	BackupWithTags.Spec.Tags = []string{backupTag}
+	return BackupWithTags
+}
+
+func (ts *BackupTestSuite) assertJobHasTagArguments(job *batchv1.Job) {
+	JobArguments := job.Spec.Template.Spec.Containers[0].Args
+	found := false
+	for i := range JobArguments {
+		if JobArguments[i] == backupTag {
+			found = true
+			break
+		}
+	}
+	ts.Assert().True(found, "did not find tag in job argmuments")
```

Comment:
I think there's a `ts.Assert().Contains` function that you can use to check the existence of an element in a string slice.
```suggestion
	jobArguments := job.Spec.Template.Spec.Containers[0].Args
	ts.Assert().Contains(jobArguments, backupTag, "backup tag in job args")
```